src/BuildExecutable.jl

```diff
@@ -41,16 +41,21 @@ type SysFile
     inference0
 end
 
-function SysFile(exename)
-    buildpath = abspath(dirname(Libdl.dlpath("libjulia")))
+function SysFile(exename, debug=false)
+    buildpath = abspath(dirname(Libdl.dlpath(debug ? "libjulia-debug" : "libjulia")))
     buildfile = joinpath(buildpath, "lib"*exename)
     inference = joinpath(buildpath, "inference")
     inference0 = joinpath(buildpath, "inference0")
     SysFile(buildpath, buildfile, inference, inference0)
 end
 
-function build_executable(exename, script_file, targetdir=nothing, cpu_target="native"; force=false)
-    julia = abspath(joinpath(JULIA_HOME, "julia"))
+function build_executable(exename, script_file, targetdir=nothing, cpu_target="native";
+                          force=false, debug=false)
+    julia = abspath(joinpath(JULIA_HOME, debug ? "julia-debug" : "julia"))
+    if !isfile(julia)
```

Comment:
I think the failure stems from here, it should be `isfile(julia*@windows? ".exe" : "")` as `isfile` needs the full file name.